lib/rules/prop-types.js

```diff
@@ -164,6 +164,16 @@ module.exports = function(context) {
     return tokens.length && tokens[0].value === '...';
   }
 
+  function getKeyValue(node) {
+    var key = node.key;
+    if (key) {
+      if (key.type === 'Identifier') {
+        return key.name;
+      }
+      return key.value;
+    }
```

Comment:
why not 
```js
return (key.type === 'Identifier') ? key.name : key.value;
```

here https://github.com/yannickcr/eslint-plugin-react/pull/123/files#diff-6149d4075a6f3125f18ce8c87bb253cbR39 you've done it like that:
```js
return node.key.type === 'Identifier' ? node.key.name : node.key.value;
```

Of course it is just a matter of taste :cherries: